test/test-license.rb

```diff
@@ -0,0 +1,24 @@
+class LicenseTest < Test::Unit::TestCase
```

Comment:
Nothing below this line has been changed in this file.

Reply:
We can remove this file.